modules/jdk.packager/src/main/native/library/common/PosixPlatform.cpp

```diff
@@ -53,6 +53,7 @@
 #include <algorithm>
 #include <dlfcn.h>
 #include <signal.h>
+#include <sys/wait.h>
```

Comment:
Maybe it is better to put this within "#ifdef LINUX" ?

Reply:
I don't know, but this is a standard POSIX header, so I guess any os/arch building PosixPlatform.cpp is likely to support it.

Reply:
No need to add ifdef, it is good